RobotCode/src/main/java/frc/robot/subsystems/CdTraySubsystem.java

```diff
@@ -27,21 +27,11 @@ public CdTraySubsystem()
 
 		mainCompressor.enableDigital();
 
-		cdArmLeft.set(Value.kOff);
-		cdArmRight.set(Value.kOff);
```

Comment:
where did all dem koff go

Reply:
Owen told us to killing them. Just following orders.

Reply:
the funny man with the lightning actually has a good point